.githooks/pre-commit/lint

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+npm run lint
```

Comment:
I'm against stuff like this: https://github.com/aeternity/aepp-identity/pull/82

Reply:
I would like to catch most of the issues before committing and/or pushing.

Reply:
So, I'm repeating some points from https://github.com/aeternity/aepp-identity/pull/82:
- it is not working properly when developer trying to commit part of the changes of the working directory
- It slows down git performance
- these checks are already performs in CI